lib/mix/lib/mix/tasks/escript.build.ex

```diff
@@ -159,10 +161,12 @@ defmodule Mix.Tasks.Escript.Build do
     # use default true if neither are present.
     #
     # TODO: Deprecate :strip_beam option on v1.13
-    strip_beams? =
-      Keyword.get_lazy(escript_opts, :strip_beams, fn ->
+    {strip_beams?, chunks_to_keep} =
+      escript_opts
+      |> Keyword.get_lazy(:strip_beams, fn ->
         Keyword.get(escript_opts, :strip_beam, true)
       end)
+      |> Mix.Release.parse_strip_beams_option()
```

Comment:
Yeah i agree, it also felt a bit weird to me, tbh :)

Reply:
@fertapric The only part i feel bad about is that, in that case,  i probably have to move the parsing of the option to inside `Mix.Release.parse_strip_beams_option`, which will effectively run it for each BEAM :(

Reply:
@noozo I see 🤔   Then maybe we can keep `parse_strip_beams_option` public, but mark it with `@doc false` though.

Or maybe duplicate that function in both modules since the logic is quite simple. Both sound good to me.